hdc/algo/accessors.py

```diff
@@ -134,6 +136,57 @@ class DekadPeriod(Period):
     _period_cls = Dekad
 
 
+@xarray.register_dataset_accessor("season")
+@xarray.register_dataarray_accessor("season")
+class SeasonPeriod(AccessorTimeBase):
```

Comment:
As we no longer inherit from Period, perhaps the class should be called something without Period